### ISSUE TITLE
Not all theme styles are respected

### ISSUE THREAD

**Issue by esskar** [codegrue/card_settings#112]
Hello,

first, kudos to this awesome library. it helps a lot.

I just noticed that not all theme styles are respected. I set the default font familiy to Calibri Light, but the value text is still set to .SF UI Text for Cupertino

![image](https://user-images.githubusercontent.com/65206/84808880-99380100-b009-11ea-8f52-5076442ac40d.png)

**Comment by esskar** [comment]
ah, I am using [flutter_platform_widgets](https://pub.dev/packages/flutter_platform_widgets) and building a cupertino app for ios; so theme is not same as theme here

**Comment by codegrue** [comment]
Still the text theme should be honored. I assume this is a bug.

**Comment by codegrue** [comment]
Adding this to the ThemeData seems to work fine globally. Closing unless you are able to post code verifying the bug.

``` dart
      return ThemeData(
        ...
        textTheme: GoogleFonts.paprikaTextTheme(),
        ...
      );
```

**Comment by esskar** [comment]
Its an issue with cupertino apps

```dart
class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return CupertinoApp(
        title: 'Flutter Demo',
        theme: CupertinoThemeData(
          primaryColor: Colors.black,
          textTheme: CupertinoTextThemeData(textStyle: GoogleFonts.pacifico()),
        ),
        home: CupertinoPageScaffold(
          navigationBar: CupertinoNavigationBar(
            middle: Text("Hello World"),
          ),
          child: Form(
            child: CardSettings(
              children: [
                CardSettingsSection(
                  children: <Widget>[
                    CardSettingsText(
                      label: 'Title',
                      initialValue: "No title yet",
                    ),
                    CardSettingsSelectionPicker(
                      label: "Choose wisely",
                      options: ["Foo", "Bar"],
                      initialValue: "Foo",
                    )
                  ],
                )
              ],
            ),
          ),
        ));
  }
}
```

![image](https://user-images.githubusercontent.com/65206/85108486-15039a80-b210-11ea-81bf-ff25340b6a04.png)

**Comment by esskar** [comment]
and the new example looks like that

![image](https://user-images.githubusercontent.com/65206/85129740-eeeff180-b233-11ea-9bc6-a4446d244e7a.png)

**Comment by esskar** [comment]
but, i just found, that i can wrap my `CupertinoApp` in an `Theme,` like

```dart
Theme(
   data: materialThemeData,
   child: CupertinoApp(...)
)
```

this seems to work

**Comment by codegrue** [comment]
Either the global theme or a local theme (like your solution) should work. I verified the Cupertino controls honor the font so keeping closed. This is fixed in release 1.9.8
